lib/core/backend/backend.ts

```diff
@@ -298,7 +297,7 @@ export class Backend {
     }
 
     this._kuzzle = new Kuzzle(this.config.content);
-
+    this.log = this._kuzzle.log.child(`app:${this.name}`);
```

Comment:
```suggestion
    this.log = this._kuzzle.log.child(`app`);
```
I would just create the "app" level and let the developper add its how sublevel as he wishes